tests/baselines/reference/objectSpreadSetonlyAccessor.types

```diff
@@ -26,14 +26,14 @@ const o2: { foo: undefined } = { foo: 1, ... { set foo(_v: number) { } } }
 >   : ^^^^^^^         ^^^
 >foo : undefined
 >    : ^^^^^^^^^
->{ foo: 1, ... { set foo(_v: number) { } } } : { foo: undefined; }

```

Comment:
...this just seems wrong? The `set` in the spread 100% overrides the `1`, so pulling the type from the `1` leads to an incorrect declaration. Another case where there being multiple declarations for the symbol maybe implies it's not safe to just reuse the type from the initializer!

Reply:
Reverted to old behavior.